### ISSUE TITLE
make dataset self-contained

### ISSUE THREAD

**Issue by wangleiphy** [wangleiphy/jax-flow-matching#1]
like this

```python
data = np.load('data.npz')
L = data["L"] 
x = data["position"] 
```

**Comment by wangleiphy** [comment]
now `L` was set by hand https://github.com/wangleiphy/jax-flow-matching/blob/lj/src/main.py#L45

**Comment by wangleiphy** [comment]
```python
import numpy as np
data = np.load('../data/LJSystem_npz/liquid/traj_N32_rho0.7_T1.0.npz')
print (data.files)

['positions', 'N', 'T', 'density', 'cell_vectors', 'reduced_T', 'reduced_density', 'dt', 'm', 'sigma', 'epsilon']
```

**Comment by wangleiphy** [comment]
![image](https://user-images.githubusercontent.com/1732299/213134104-21d56fb2-8647-436f-9d3e-d1224eba4b0a.png)

**Comment by wangleiphy** [comment]
closed by dd450363507dd9754cfab43b7895ba42467d9301
